README.md

```diff
@@ -1,6 +1,12 @@
 # Listable
 
-This Neos package solves one problem: help you list any nodes in TypoScript.
+For the neos user interface, this package provides a content element "List" (experimental) that can render a list of nodes with a ListableMixin that can be added to node types that you want to list.
+
+As a website creator, you usually simply render lists with one Fluid template that has a ``for`` statement that iterates over a list item collection you passed to it (best MVC practice). https://neos.readthedocs.org/en/stable/CreatingASite/RenderingCustomMarkup/Templating.html
```

Comment:
Using `<f:for>` is a kind of anti-pattern. Most of us usually use TYPO3.TypoScript:Collection object when rendering lists of things.
I use `<f:for>` in this package just because of pagination widget.